admin-panel/src/main/java/net/cryptic_game/backend/admin/exception/InternalServerErrorException.java

```diff
@@ -0,0 +1,12 @@
+package net.cryptic_game.backend.admin.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class InternalServerErrorException extends ResponseStatusException {
+
+    public InternalServerErrorException(final String message) {
+        super(HttpStatus.INTERNAL_SERVER_ERROR, message);
+    }
+
+}
```

Comment:
You don't need this, because Spring Web Flux returns 500 if there is any exception thrown in the execution.